tests/unit-tests/attributes/functions.php

```diff
@@ -202,4 +206,19 @@ public function test_wc_delete_attribute() {
 		$result = wc_delete_attribute( 9999999 );
 		$this->assertFalse( $result );
 	}
+
+	/**
+	 * Test counts of attributes.
+	 */
+	public function test_count_attribute_terms() {
+		$global_attribute_data = WC_Helper_Product::create_attribute( 'test', array( 'Chicken', 'Nuggets' ) );
```

Comment:
Thanks for this test @mikejolley - I was _close_ to having a test like this, but in many many many more lines of code, so really nice for me to see it done this way here.